cursorless-talon/src/spoken_scope_forms.py

```diff
@@ -4,25 +4,20 @@
 
 
 def init_scope_spoken_forms(graphemes_talon_list: dict[str, str]):
-    create_flattened_talon_list(
-        csv_get_ctx(), graphemes_talon_list, include_custom_regex=True
-    )
+    create_flattened_talon_list(csv_get_ctx(), graphemes_talon_list)
     if is_cursorless_test_mode():
-        create_flattened_talon_list(
-            csv_get_normalized_ctx(), graphemes_talon_list, include_custom_regex=False
-        )
+        create_flattened_talon_list(csv_get_normalized_ctx(), graphemes_talon_list)
 
 
-def create_flattened_talon_list(
-    ctx: Context, graphemes_talon_list: dict[str, str], include_custom_regex: bool
-):
+def create_flattened_talon_list(ctx: Context, graphemes_talon_list: dict[str, str]):
     lists_to_merge = {
         "cursorless_scope_type": "simple",
         "cursorless_surrounding_pair_scope_type": "surroundingPair",
         "cursorless_selectable_only_paired_delimiter": "surroundingPair",
         "cursorless_wrapper_selectable_paired_delimiter": "surroundingPair",
     }
-    if include_custom_regex:
+    # If the user have no custom regex scope type, then that list is missing from the context
+    if "user.cursorless_custom_regex_scope_type" in ctx.lists.keys():  # noqa: SIM118
```

Comment:
This is really weird. ruff wants me to remove `.keys()` but if I do this condition is always false. I wonder if this has anything to do with how Talon implements the lists/dict?

Reply:
yeah SIM118 is fairly aggressive imo. Fine with a noqa but if it keeps biting us we might consider just shutting it off

Reply:
I think it should be equivalent generally in python and this is something Talon is doing